app/test_main.py

```diff
@@ -1,3 +1,72 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,result",
+        [
+
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="should calculate first year if < 15"
+            ),
+            pytest.param(
+                15,
+                15,
+                [1, 1],
+                id="should calculate first year"
+            ),
+            pytest.param(
+                23,
+                23,
+                [1, 1],
+                id="should calculate second year"
+            ),
+            pytest.param(
+                28,
+                28,
+                [3, 2],
+                id="should calculate third year differently"
+            ),
```

Comment:
what about 24 for both or 27 for a cat and 29 for a dog?) You should check all age boundaries